test/REQUIRE

```diff
@@ -1,3 +1,3 @@
-JuMP
+JuMP 0.16
```

Comment:
But in the Travis run after your commit, it actually uses JuMP 0.15.1, as it did on my local machine.